frontend/src/pages/PickupGem/PickupGem.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <router-view :name="currentStageName" :gemPackage="gemPackage" @nextStage="nextStage"></router-view>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import dayjs from "dayjs"; 
+
+// eslint-disable-next-line no-unused-vars
+enum PickupGemStage {
+  // eslint-disable-next-line no-unused-vars
+  FoundGem,
+  // eslint-disable-next-line no-unused-vars
+  MessageDisplay,
+}
+
+export default defineComponent({
+  setup() {
+    const gemPackage = {
```

Comment:
nit/feel free ignore: maybe can add a TODO here to replace it with fetched data or props

Reply:
Okay!